src/tests/ie_test_utils/common_test_utils/ov_tensor_utils.cpp

```diff
@@ -192,6 +192,14 @@ void compare(const ov::Tensor& expected,
 
     auto expected_data = expected.data<ExpectedT>();
     auto actual_data = actual.data<ActualT>();
+
+    if (getenv("DUMP_OUTPUTS")) {
+        for (auto i = 0; i < shape_size(actual_shape); i++) {
+            std::cout << "index " << i << ": " << expected_data[i] << " <-> " << actual_data[i] << std::endl; 
+        }
+        //return;
+    }
+
```

Comment:
Better to implement in scope of separate PR if it is supposed to be kept

Reply:
removed.